docs/installation.md

```diff
@@ -0,0 +1,109 @@
+# Install Valkey from Percona repositories
+
+Valkey offers packages for a select number of Linux distributions. Specifically, you can install it from Fedora/EPEL yum repositories and as binary tarballs for both Ubuntu Bionic and Ubuntu Focal. Additionally, it is available as a Docker container.
```

Comment:
and several of RHEL derivatives: [AlmaLinux](https://almalinux.org/), [Rocky Linux](https://rockylinux.org/)

Reply:
Does it relate to packages?

Reply:
Updated